### PR TITLE
Added sanitize parameter for Text widget

### PR DESCRIPTION
**Pull request by spamsch**
Trying a fix for https://github.com/mljar/mercury/issues/303 as I need the Text widget to not sanitize sometimes.

### CODE REVIEW

**Comment by spamsch** [comment]
When trying to test these changes I cannot get the worker in the browser to load. Everything looks fine after runnin `mercury run` and then navigating to my notebook. But it does not stop loading. Seems the web worker is not initializing correctly. No error messages in the browsers console.

**Comment by spamsch** [comment]
Made more checks and it now works cleanly

File: frontend/.nvmrc
Comment (pplonski) [inline frontend/.nvmrc:1]: do we need this file?

**Review by pplonski — COMMENTED** [review]
Thank you very much for PR. It looks good. One concern about `.nvmrc` file, do we need it?

File: frontend/.nvmrc
Comment (spamsch) [reply frontend/.nvmrc:1]: I was struggling when compiling the frontend as my default node is v18. But compilation only works reliably with v16. So I added .nvmrc that acts a implicit documentation and one can just use `nvm use` before compiling.

**Review by pplonski — APPROVED** [review]

**Pull request merged**